docs/installation.rst

```diff
@@ -76,6 +76,7 @@ The following optional dependencies are required to support certain features:
 * `usd-core <https://pypi.org/project/usd-core>`_: Required for some Warp examples, ``warp.sim.parse_usd()``, and ``warp.render.UsdRenderer``.
 * `JAX <https://jax.readthedocs.io/en/latest/installation.html>`_: Required for JAX interoperability (see :ref:`jax-interop`).
 * `PyTorch <https://pytorch.org/get-started/locally/>`_: Required for PyTorch interoperability (see :ref:`pytorch-interop`).
+* `Paddle <https://www.paddlepaddle.org.cn/>`_: Required for Paddle interoperability (see :ref:`paddle-interop`).
```

Comment:
Maybe link to the GitHub instead? https://github.com/PaddlePaddle/Paddle In the US, the website doesn't seem to load.

Reply:
> Maybe link to the GitHub instead? https://github.com/PaddlePaddle/Paddle In the US, the website doesn't seem to load.

I did forget to upload paddle.py😥. It has now been uploaded along with the changes to the paddle link through `git commit --amend`.